sculture-android/app/src/main/java/tr/edu/boun/cmpe/sculture/activity/ProfilePageActivity.java

```diff
@@ -0,0 +1,161 @@
+package tr.edu.boun.cmpe.sculture.activity;
+
+/**
+ * Created by SahaOperasyon2 on 06.12.2015.
+ */
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import tr.edu.boun.cmpe.sculture.R;
+import tr.edu.boun.cmpe.sculture.adapter.StoryListViewAdapter;
+import tr.edu.boun.cmpe.sculture.models.response.BaseStoryResponse;
+import tr.edu.boun.cmpe.sculture.models.response.SearchResponse;
+
+import static tr.edu.boun.cmpe.sculture.BaseApplication.baseApplication;
+import static tr.edu.boun.cmpe.sculture.Constants.API_USER_STORIES;
+import static tr.edu.boun.cmpe.sculture.Constants.FIELD_ID;
+import static tr.edu.boun.cmpe.sculture.Constants.FIELD_PAGE;
+import static tr.edu.boun.cmpe.sculture.Constants.FIELD_SIZE;
+import static tr.edu.boun.cmpe.sculture.Constants.PREF_USER_ID;
+import static tr.edu.boun.cmpe.sculture.Utils.addRequest;
+
+public class ProfilePageActivity extends AppCompatActivity {
+
+    TextView username;
+    TextView email;
+    RecyclerView story_list_recycler;
+    RelativeLayout loggedInLayout;
+    Button follow;
+
+    LinearLayoutManager mLayoutManager;
+    StoryListViewAdapter mStoryListViewAdapter;
+
+    private int PAGE = 1;
+    private boolean is_loading_more = false;
+    private boolean is_reach_end = false;
+
+    long currentUserID;
+    long visitedUserID;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_profile_page);
+
+
+        loggedInLayout = (RelativeLayout) findViewById(R.id.profile_layout_loggedIn);
+
+        username = (TextView) findViewById(R.id.profile_username);
+        email = (TextView) findViewById(R.id.profile_email);
+
+        follow = (Button) findViewById(R.id.follow);
+
+        story_list_recycler = (RecyclerView) findViewById(R.id.profile_story_list);
+        mLayoutManager = new LinearLayoutManager(this);
+        mStoryListViewAdapter = new StoryListViewAdapter(this);
+        story_list_recycler.setLayoutManager(mLayoutManager);
+        story_list_recycler.setAdapter(mStoryListViewAdapter);
+
+        setRecyclerListeners();
+
+        load_story();
```

Comment:
You should place this method after a point the user profile is retrieved. For example response listener of the API /user/get